modules/ossm-about-directing-egress-traffic-through-a-gateway.adoc

```diff
@@ -0,0 +1,9 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-about-directing-egress-traffic-through-a-gateway_{context}"]
+= About directing egress traffic through a gateway
+:context: ossm-about-directing-egress-traffic-through-a-gateway
```

Comment:
Is there a reason that you're setting the context variable here? That normally should only be done in the assembly

Reply:
Yup...the reason is I made a mistake. Thanks for caching it!

Reply:
done